### PR TITLE
Docs: rework readme

### PR DESCRIPTION
**Pull request by stsewd**
- Fix some typos
- Remove some unnecessary text that was taking space
- Use double quotes in lua examples (this seems to be the convention in
  the source code)
- Sort list of languages, so they are easy to find.

Also moved the image after the title, it felt weird having the title below

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,77 +45,55 @@ $ git clone https://github.com/nvim-treesitter/nvim-treesitter.git
...
+- `TSInstall {language}` to install one or more parsers.
+  `TSInstall <tab>` will give you a list of supported languages, or select `all` to install them all.
+- `TSInstallInfo` to know which parser is installed.
 
-```vim
-:TSInstall c
-Downloading...
-Compiling...
-Treesitter parser for c has been installed
+Let's say you need parsers for `lua`, this is how you do with these commands:
```
Comment (stsewd) [inline README.md:55]: This example was taking a lot of space before having the configuration, so I cut it a little.

File: README.md [hunk 2]
```diff
@@ -146,10 +125,10 @@ require'nvim-treesitter.configs'.setup {
...
     keymaps = {
         ["iL"] = { -- you can define your own textobjects directly here
-        python = "(function_definition) @function",
-        cpp = "(function_definition) @function",
-        c = "(function_definition) @function",
-        java = "(method_declaration) @function"
+          python = "(function_definition) @function",
+          cpp = "(function_definition) @function",
+          c = "(function_definition) @function",
+          java = "(method_declaration) @function"
```
Comment (stsewd) [inline README.md:131]: Sorry, missed this indentation in the morning

**Comment by stsewd** [comment]
Next step on my list is having one section per module instead of having a big one :memo:

File: README.md [hunk 3]
```diff
@@ -310,15 +290,15 @@ Before doing anything run `:checkhealth nvim_treesitter`. This will help you fin
 
 ### Feature `X` does not work for language `Y`...
```
Comment (vigoux) [inline README.md:291]: Maybe you should change `Y` to `{language}` here too.

File: README.md [hunk 4]
```diff
@@ -264,39 +243,40 @@ For `nvim-treesitter` to work, we need to use query files such as those you can
...
 - [ ] vue
-- [ ] elm
 - [ ] yaml
-- [ ] nix
-- [ ] markdown
-- [x] regex (maintained by @theHamsta)
-- [ ] jsdoc
-- [x] dart (maintained by @Akin909)
-- [x] rst (maintained by @stsewd)
-- [x] tree-sitter query language (maintained by @steelsojka)
```
Comment (vigoux) [inline README.md:280]: The rendered version looks weird here, but I don't know if that comes from the previewer or not.

Comment (vigoux) [reply README.md:280]: Was the previewer's fault

**Comment by vigoux** [comment]
Just rebase this and we will be good to go !

**Pull request merged**